Riot/Modules/Common/ActivityIndicator/FullscreenActivityIndicatorPresenter.swift

```diff
@@ -0,0 +1,53 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import CommonKit
```

Comment:
I don't particularly like the idea of these files having a dependency on CommonKit, that target doesn't add anything. We should import them directly into element.

Reply:
As discussed elsewhere, keeping the import and modularized code